app/models/wiki-variables.js

```diff
@@ -35,10 +35,22 @@ WikiVariablesModel.reopenClass({
 
 				if (contentType && contentType.indexOf('application/json') !== -1) {
 					return response.json();
-				} else {
+				} else if (url !== response.url) {
+					// API was redirected to non-json page
 					throw new NonJsonApiResponseError().withAdditionalData({
```

Comment:
This error's name doesn't fit anymore. Below we have `// non-json API response` and it's not used there. How about `WikiVariablesRedirectError`?